code/game/objects/items/contraband.dm

```diff
@@ -16,6 +16,31 @@
 	desc = "Highly illegal drug. Bang - and your stress is gone."
 	starts_with = list(/obj/item/reagent_containers/pill/joy = 3)
 
+/obj/item/storage/pill_bottle/heroin
+	name = "bottle of Heroin pills"
+	desc = "Highly illegal drug. For quick pain removal."
+	starts_with = list(/obj/item/reagent_containers/pill/heroin = 3)
+
+/obj/item/storage/pill_bottle/cocaine
+	name = "bottle of Cocaine tablets"
+	desc = "Supposedly a highly illegal drug... yet the labeling on the bottle is suspiciously perfect..."
+	starts_with = list(/obj/item/reagent_containers/pill/cocaine = 5)
+
+/obj/item/storage/pill_bottle/contemplus
+	name = "bottle of Contemplus tablets"
+	desc = "A Yomi Genetics bottle clearly marked as 'for animal testing only.' You doubt this is followed often on Venus..."
+	starts_with = list(/obj/item/reagent_containers/pill/contemplus = 5)
+
+/obj/item/storage/pill_bottle/spotlight
+	name = "bottle of Spotlight tablets"
+	desc = "A Zavodskoi bottle with a conspicuous 'defective' stamp on it. You doubt this was actually defective."
+	starts_with = list(/obj/item/reagent_containers/pill/spotlight = 5)
+
+/obj/item/storage/pill_bottle/sparkle
+	name = "bottle of Sparkle tablets"
+	desc = "A Zeng-Hu bottle clearly marked as being for 'medical testing purposes only.' As if..."
+	starts_with = list(/obj/item/reagent_containers/pill/sparkle = 5)
+
 /obj/item/storage/pill_bottle/smart
 	name = "bottle of Smart pills"
 	desc = "Highly illegal drug. For exam season."
```

Comment:
I am unsure if we should label all of them that clearly.

Reply:
Heroin is consistent with the others.

Cocaine, that's to reflect Venus lore, where it is bought and sold on a regular basis. The idea here is that the various medical companies cover themselves (since I highly doubt this is legal by Sol law) by thinly claiming it's not for human consumption.